source/renderer/app/components/wallet/settings/WalletPublicKeyQRCodeDialog.scss

```diff
@@ -1,24 +1,14 @@
 .dialog {
   :global {
-    .Dialog_title {
-      margin-bottom: 0;
+    .Dialog_contentWrapper::-webkit-scrollbar-track {
```

Comment:
@aleksandardjordjeviciohk shouldn't we put this into an override for the Dialog instead of the global here? 🤔

Reply:
This is obsolete now, as it was a leftover from previous implementation. Deleted now.